.circleci/config.yml

```diff
@@ -2,19 +2,20 @@
 
 version: 2
 references:
-  container_python36: &container_python36
+  container_python310: &container_python310
     docker:
-      - image: circleci/python:3.6.2
+      - image: cimg/python:3.10.11
       - name: localstack
-        image: localstack/localstack
-    working_dir: ~/project
+        image: localstack/localstack:2.0.2
+    working_directory: ~/repo
 
   container_python38: &container_python38
     docker:
       - image: circleci/python:3.8.1
       - name: localstack
-        image: localstack/localstack
-    working_directory: ~/project
+        image: localstack/localstack:2.0.2
```

Comment:
Should it still be localstack/localstack?

Reply:
this has been brought into line with other (cma-python) circle-ci configs during debugging of an error

Reply:
https://github.com/nasa/cumulus-message-adapter-python/blob/master/.circleci/config.yml#L18
It uses localstack/localstack for 3.8.1.
From the ci build log of cma-python, the current LocalStack version: 2.1.1.dev
It's fine to pin the version to 2.0.2 if the latest doesn't work? for 3.10.
Otherwise use the latest is better.